apps/web/src/pages/auth/QuestionnairePage.tsx

```diff
@@ -3,9 +3,12 @@ import AuthContainer from '../../components/layout/components/AuthContainer';
 import { QuestionnaireForm } from './components/QuestionnaireForm';
 import { useVercelIntegration } from '../../hooks';
 import SetupLoader from './components/SetupLoader';
+import { ENV, IS_DOCKER_HOSTED } from '@novu/shared-web';
+import { HubspotSignupForm } from './components/HubspotSignupForm';
 
 export default function QuestionnairePage() {
   const { isLoading } = useVercelIntegration();
+  const isNovuProd = !IS_DOCKER_HOSTED && ENV === 'production';
```

Comment:
Nice one, I think this is a suitable approach to dealing with local development, it will ensure we serve the Questionnaire form that does not require an internet connection in local.

For the no internet case in Prod, the User would not be able to reach the Novu API to complete the Organization creation anyway, so no further action needed on this path.